test/TestUtilities/Test.Utility/ProtocolUtility.cs

```diff
@@ -26,7 +27,7 @@ public static string GetResource(string name, Type type)
 
         public static string CreateServiceAddress()
         {
-            return string.Format("http://{0}/", Guid.NewGuid());
+            return string.Format(CultureInfo.CurrentCulture, "http://{0}/", Guid.NewGuid());
```

Comment:
It looks like an URL with an invariant culture parameter

```suggestion
            return string.Format(CultureInfo.InvariantCulture, "http://{0}/", Guid.NewGuid());
```